### PR TITLE
Implement PVR API 5.1.0 

### PR DESCRIPTION
**Pull request by manuelm**
This is a minimal implementation for PVR API 5.1.0 (+ some older commits I had lying around)

There's still a lot to do but this should at least get the PVR working again

For yet unknown reason Kodi doesn't call `CloseLiveStream` + `OpenLiveStream` on channel switching anymore. So the PVR is doing this now.

Another issue is the video players current time is jumping back and forth as soon as I'm returning `false` on `IsRealTimeStream`. Although there is at least 10MB of data (I'm not doing any demuxing) in the local timeshift buffer available. So for now I'm just returning `true` here.

### CODE REVIEW

File: pvr.dvbviewer/addon.xml.in [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.dvbviewer"
-  version="2.0.1"
+  version="2.0.2"
```
Comment (ksooo) [inline pvr.dvbviewer/addon.xml.in]: For the other add-ons, we did a major version bump for Krypton.

Comment (manuelm) [inline pvr.dvbviewer/addon.xml.in]: There was one which bumped from 1.11.10 to 2.0.0. But it's probably better to bump to 3.1.0 here. thank you

File: pvr.dvbviewer/addon.xml.in [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.dvbviewer"
-  version="2.0.1"
+  version="3.1.0"
```
Comment (ksooo) [inline pvr.dvbviewer/addon.xml.in]: It's not very important, but you were right. The major bump was  2.0.0 Soy if we do it like the other add-ons, 2.1.0 would be correct here.

Comment (manuelm) [inline pvr.dvbviewer/addon.xml.in]: arr my fault again. I thought 2.1.0 but typed 3.1.0.. I guess it's time for some break.

**Comment by ksooo** [comment]
Maybe you find this useful? https://github.com/ksooo/pvr.dvbviewer/commits/pvr-api-5-0-0

**Comment by manuelm** [comment]
Actually I already used most of your commits in the various PVRs as reference and to fill my local TODO list.

**Comment by ksooo** [comment]
API changes implementation look okay to me. +1 for this from my side.
Have not reviewed the other changes, though.

**Comment by manuelm** [comment]
Will merge this now and create a PR for enabling the PVR again. Thank you @ksooo

**Pull request merged**